### PR TITLE
Feature/header

### PR DESCRIPTION
**Pull request by Olujuwon**
- Header Implemented with Language dropdown

- No mobile Design

### CODE REVIEW

File: src/components/App/App.css [hunk 1]
```diff
@@ -0,0 +1,5 @@
+.App {
+  text-align: center;
+  width: 80%;
+  margin: auto;
+}
```
Comment (jimppa) [inline src/components/App/App.css]: Unused file, remove.

File: src/components/Header/Header.js [hunk 2]
```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import propTypes from 'prop-types';
```
Comment (jimppa) [inline src/components/Header/Header.js:2]: I don't see prop-types listed in package.json

File: src/components/Header/Header.js [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+    this.setState({
+      dropdownOpen: !this.state.dropdownOpen,
+    });
+  }
+
+  render(){
+    return (
+      //The leftside starts here, Helsink Logo and the title
+      <div className={styles.headerContainer}>
+        <div className={styles.logo}>
```
Comment (jimppa) [inline src/components/Header/Header.js]: Helsinki logo should be an image instead of background

File: src/index.scss [hunk 4]
```diff
@@ -0,0 +1,5 @@
+body {
+  margin: 0;
+  padding: 0;
+  font-family: sans-serif;
+}
```
Comment (jimppa) [inline src/index.scss]: This file is not used and should be removed.

File: src/components/Landingpage/landing.scss [hunk 5]
```diff
@@ -5,11 +5,13 @@
...
  
 .koroBg:before {
-  height: 100px;
+  width:100%;
+  height: 200px;
   content: '';
   display: block;
-  @include koro('beat', $hel-metro, 500);
+  @include koro('pulse', $hel-metro, 465);
 }
```
Comment (jimppa) [inline src/components/Landingpage/landing.scss]: Changing landing page styles should be in their own pull request.

File: src/components/Header/Header.js [hunk 6]
```diff
@@ -0,0 +1,79 @@
...
+  }
+}
+
+Header.propTypes = {
+  logoTitle: propTypes.string,
+  leftIcon: propTypes.string,
+  languages: propTypes.array,
+};
+
+export default Header;
```
Comment (jimppa) [inline src/components/Header/Header.js]: - What is the point of the leftIcon prop? If it's set then both language switcher and login has the same icon.
- There is only one header on the page so it's not important that everything in it can be changed by props.

File: src/components/Header/Header.js [hunk 7]
```diff
@@ -0,0 +1,79 @@
...
+              </DropdownToggle>
+              <DropdownMenu>
+                {this.props.languages.map((lang, index)=> <div 
+                  className={styles.langDiv }
+                  onClick={this.toggle}
+                  key={index}>{lang}</div>)}
+              </DropdownMenu>
+            </Dropdown>
+          </div>
+          <div className={styles.iconContainer}>
```
Comment (jimppa) [inline src/components/Header/Header.js]: Signup does not need login functionality in header, this section can be removed.

File: src/components/Header/header.scss [hunk 8]
```diff
@@ -0,0 +1,55 @@
...
+}
+
+.langDiv{
+    width: auto;
+}
+
+.langDiv:hover{
+    color: $hel-white;
+    background-color: $hel-dark;
+}
```
Comment (jimppa) [inline src/components/Header/header.scss:51]: 2 spaces indentation as in rest of application.

File: src/scss/helsinki/_custom-styles.scss [hunk 9]
```diff
@@ -57,6 +57,7 @@ $input-form-btn-padding-x-sm:    0.5rem;
 
 $input-form-btn-padding-x-lg:    1rem;
 
+$line-height-computed: 1rem;
```
Comment (jimppa) [inline src/scss/helsinki/_custom-styles.scss]: Don't really see the point in this variable, could be deleted.

**Review by jimppa — CHANGES_REQUESTED** [review]
Please fix the mentioned issues.
Also the styles do have some issues:
- Layout breaks on small screen (text goes on top of logo on the left side).
- The header-items should have some padding near the edges, see layout.

File: src/components/Header/Header.js (hunk 2 above)
Comment (Olujuwon) [reply src/components/Header/Header.js:2]: 'prop-types' is inbuilt react's implementation to try to address type-checking, it comes with CRA .

File: src/components/Header/header.scss (hunk 8 above)
Comment (Olujuwon) [reply src/components/Header/header.scss:51]: The spacing is according to eslint config.
